server/events/command_runner.go

```diff
@@ -416,6 +416,10 @@ func (c *DefaultCommandRunner) updatePull(ctx *CommandContext, command PullComma
 	if err := c.VCSClient.CreateComment(ctx.BaseRepo, ctx.Pull.Num, comment); err != nil {
```

Comment:
I couldn't get the local Docker env and makefiles running correctly so I wasn't able to regenerate mocks. If someone with a working dev env could do that, it'd be helpful.